src/types/locale.ts

```diff
@@ -0,0 +1,59 @@
+import { formatValue } from "../helpers";
+import { Alias } from "./misc";
+
+export type LocaleCode = string & Alias;
+
+export interface Locale {
+  name: string;
+  code: string;
+  thousandsSeparator: string;
+  decimalSeparator: string;
+  dateFormat: string;
+  timeFormat: string;
+}
+
+export const DEFAULT_LOCALES: Locale[] = [
+  {
+    name: "United States",
+    code: "en-US",
+    thousandsSeparator: ",",
+    decimalSeparator: ".",
+    dateFormat: "m/d/yyyy",
+    timeFormat: "hh:mm:ss a",
+  },
+  {
+    name: "France",
+    code: "fr-FR",
+    thousandsSeparator: " ",
+    decimalSeparator: ",",
+    dateFormat: "dd/mm/yyyy",
+    timeFormat: "hh:mm:ss",
+  },
+];
+export const DEFAULT_LOCALE: Locale = DEFAULT_LOCALES[0];
+
+export function isValidLocale(locale: any): locale is Locale {
```

Comment:
this function doesn't look at its place in the `types` folder :wink: 
Should probably be moved into `helpers`